README.md

```diff
@@ -231,6 +231,49 @@ models:
 
 ```
 
+#### mutually_exclusive_ranges ([source](macros/schema_tests/test_mutually_exclusive_ranges.sql))
+This test confirms that for a given lower_bound_column and upper_bound_column,
+the ranges of between the lower and upper bounds do not overlap with the ranges
+of another row.
+
+**Usage:**
+```yaml
+version: 2
+
+models:
+  # test that age ranges do not overlap
+  - name: age_brackets
+    tests:
+      - dbt_utils.mutually_exclusive_ranges:
+          lower_bound_column: min_age
+          upper_bound_column: max_age
+          collectively_exhaustive: true
```

Comment:
Is collectively exhaustive the right word here?

Reply:
in the docs below, you write:
> (i.e. there are gaps)

What do you think about `allow_gaps`? That would be the converse of what this flag currently does, but I think it would be pretty clear!